infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/DraftableObjectDao.kt

```diff
@@ -44,7 +44,7 @@ interface IDraftableObjectReader<T : Draftable<T>> {
     fun fetch(version: RowVersion<T>): T
 
     fun fetchChangeTime(): Instant
-    fun fetchDraftableChangeInfo(id: IntId<T>): DraftableChangeInfo
+    fun fetchDraftableChangeInfo(id: IntId<T>, publishType: PublishType): DraftableChangeInfo?
```

Comment:
Tässä pihviä numero 2: `DraftableChangeInfo` sisältää siis luonti- ja muutosajat. Tehty tästä `publishType`:tetty haku ja sen rakennetta on muutenkin jumpattu (se sisältää enää vaan `created`- ja `changed`-aikaleimat. `changed` sisältää publishTypetetysti joko virallisen version uusimman muutosajan tai uusimman draftin muutosajan (tai siitä huolimatta virallisen muutosajan, this be a little bit complicated. Logiikka selitetty alempana))